static/js/lib/ckeditor/CKEditor.ts

```diff
@@ -22,19 +19,16 @@ import MarkdownMediaEmbed from "./plugins/MarkdownMediaEmbed"
 export const FullEditorConfig = {
   plugins: [
     EssentialsPlugin,
-    UploadAdapterPlugin,
     AutoformatPlugin,
     BoldPlugin,
     ItalicPlugin,
     UnderlinePlugin,
     BlockQuotePlugin,
-    EasyImagePlugin,
     HeadingPlugin,
     ImagePlugin,
     ImageCaptionPlugin,
     ImageStylePlugin,
     ImageToolbarPlugin,
-    ImageUploadPlugin,
```

Comment:
Don't need/want these plugins?

Reply:
we will sometime soon, but in the update they added a required dependency on a configuration plugin which we don't want to set up yet, which meant that having them enabled caused a runtime error. When we go to add image uploading to the editor we'll likely re-enable these.